code/src/App.js

```diff
@@ -1,9 +1,82 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from 'react-router-dom';
+
+// Components
+import TopNav from './components/TopNav';
+import ListedMovie from './components/ListedMovie';
+import MoviePage from './pages/MoviePage';
+import { NotFound } from './pages/NotFound';
+import SimilarMovies from './pages/SimilarMovies';
+
+// API-urls
+import { ApiUrl, PosterImgUrl } from './components/ApiUrls';
```

Comment:
Capitalised names are reserved for components only, URL should rather be `MY_URL_NAME`